src/models/transfers.js

```diff
@@ -461,10 +453,26 @@ function * getFulfillment (transferId) {
   return fulfillment.getDataExternal()
 }
 
+function * resendNotification (transferId) {
```

Comment:
We already show a message in the log. Do you suggest that the ledger sends a specific error message through websocket to the client if a transfer isn't found?